test/pytorch_test_base.py

```diff
@@ -17,252 +17,279 @@
+    {
+        'test_pow_inplace_xla',  # (TPU) 0.0032 vs 0.001
+        'test_pow_inplace_3_xla',  # (TPU) 0.0028 vs 0.001
+        'test_pow_3_xla',  # (TPU) 0.0028 vs 0.001
+        'test_pow_-2_xla',  # (TPU) 0.391 vs 0.001
+        'test_topk_dim_sort_xla',  # (TPU) unimplemented HLO for X64
+        'test_topk_dim_desc_sort_xla',  # (TPU) unimplemented HLO for X64
+        'test_sort_xla',  # (TPU) unimplemented HLO for X64
+        'test_sort_neg_dim_xla',  # (TPU) unimplemented HLO for X64
+        'test_sort_neg_dim_descending_xla',  # (TPU) unimplemented HLO for X64
+        'test_sort_dim_xla',  # (TPU) unimplemented HLO for X64
+        'test_sort_dim_descending_xla',  # (TPU) unimplemented HLO for X64
+        'test_kthvalue_xla',  # (TPU) unimplemented HLO for X64
+        'test_kthvalue_neg_dim_xla',  # (TPU) unimplemented HLO for X64
+        'test_kthvalue_dim_xla',  # (TPU) unimplemented HLO for X64
+        'test_eig_with_eigvec_xla_float64',  # Precision: tensor(1.1798, dtype=torch.float64) not less than or equal to 0.001
+        'test_cumprod_xla',  # FIXME: TPU X64Rewriter doesn't support reduce-window
+        'test_cumprod_neg_dim_xla',  # FIXME: TPU X64Rewriter doesn't support reduce-window
+        'test_topk_neg_dim_sort_xla',  # (TPU) unimplemented HLO for X64
+        'test_clamp_min_xla_float64',  # float64 limit, TPU does not have real F64
+        'test_clamp_min_inplace_xla_float64',  # float64 limit, TPU does not have real F64
+        'test_clamp_max_xla_float64',  # float64 limit, TPU does not have real F64
+        'test_clamp_max_inplace_xla_float64',  # float64 limit, TPU does not have real F64
+    },
+
+    'TestTorchDeviceTypeXLA':
+    {
+        'test_cholesky_solve_batched_broadcasting',  # (TPU) 0.0039 vs 0.001
+        'test_cholesky_solve_batched_many_batches',  # (TPU) 0.36 vs 0.001
+        'test_triangular_solve_batched_many_batches',  # (TPU) 1.02 vs 0.001
+        'test_triangular_solve_batched_broadcasting',  # (TPU) 1.5 vs 0.001
+        'test_random_from_to_xla_int32',  # precision, TPU does not have real F64
+        'test_uniform_from_to_xla_float64',  # float64 limit, TPU does not have real F64
+        'test_topk_integral_xla_int64',  # (TPU) unimplemented HLO for X64
+        'test_float_to_int_conversion_finite_xla', # different behavior than numpy when casting float_max/min to int types
+    },
 
     # test_indexing.py
-    # TestIndexing
-    'test_index_put_accumulate_large_tensor_xla',  # memory limit exceeded on v2-8
+    'TestIndexingXLA':
+    {
+        'test_index_put_accumulate_large_tensor_xla',  # memory limit exceeded on v2-8
+    },
 }
 
 DISABLED_TORCH_TESTS_CPU = DISABLED_TORCH_TESTS_ANY
 DISABLED_TORCH_TESTS_GPU = DISABLED_TORCH_TESTS_ANY
+DISABLED_TORCH_TESTS_TPU = {key : DISABLED_TORCH_TESTS_ANY.get(key, set()) | DISABLED_TORCH_TESTS_TPU_ONLY.get(key, set()) for key in (DISABLED_TORCH_TESTS_ANY.keys() | DISABLED_TORCH_TESTS_TPU_ONLY.keys())}
```

Comment:
This is a very long line.
Better adding a clear API which merges two disable sets.